### ISSUE TITLE
app does not respect windows color dark mode settings

### ISSUE THREAD

**Issue by princealexi25** [smasherprog/EqTool#148]
Would love to have a dark mode.  Your app is blinding (wrt triggers window).

**Comment by smasherprog** [comment]
This is correct, themes need to be programmed, and i support just the one i have unless someone adds more
